mavlink/test_mission_ended.py

```diff
@@ -0,0 +1,123 @@
+"""
+Test for drone's destination at final waypoint by uploading mission and monitoring it.
+"""
+import sys
+import time
+
+import dronekit
+
+from modules import flight_controller
+
+
+DELAY_TIME = 10.0  # seconds
+MISSION_PLANNER_ADDRESS = "tcp:127.0.0.1:14550"
+TIMEOUT = 1.0  # seconds
+
+MAVLINK_TAKEOFF_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT
+MAVLINK_TAKEOFF_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_TAKEOFF
+MAVLINK_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT
+MAVLINK_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_WAYPOINT
+
+ALTITUDE = 10
+ACCEPT_RADIUS = 10
+
+
+def upload_mission(waypoints: "list[tuple[float, float, float]]",
```

Comment:
Add TODO that this is to be removed when the appropriate Dronekit-Python interfaces are moved from the pathing repository.